README.md

```diff
@@ -70,6 +70,30 @@ If the field references an array (again w/ or w/o nullability) of another type,
 - `@hasMany` - the foreign key is on the referenced type as `${typeName}Id`. Provide the `"as": X` argument if the name is different. (this is the reverse of `@belongsTo` in a 1-many situation).
 - `@hasAndBelongsToMany` - the foreign key on the referenced type as `${typeName}Ids`. Provide the `"as": X` argument if the name is different. (this is the reverse of `@belongsToMany` in a many-many situation).
 
+## Updating types
+
+To update types use the following command:
+
+```bash
+create-graphql-server update-type path/to/input.graphql
```

Comment:
Maybe `create-graphql-server update-type [--force-update] path/to/input.graphql`?